File: solutions/4178-vowel-consonant-score/vowel-consonant-score.py
class Solution:
    def vowelConsonantScore(self, s: str) -> int:
        v=0
        c=0
        for i in s:
            if i in 'aeiou':
                v+=1
            elif i.isalpha():
                c+=1
        return 0 if c==0 else v//c           
        

File: solutions/4178-vowel-consonant-score/vowel-consonant-score.java
class Solution {
    public int vowelConsonantScore(String s) {
        int count=0;
        String vowels="aeiou";
        int con=0;
        for(char ch:s.toCharArray()){
            if (vowels.indexOf(ch)!=-1){
                count++;
                continue;
            }
            if(Character.isAlphabetic(ch)){
                con++;
            }
        }
        return con==0?0:count/con;
    }
}